packages/ui/src/Stepper/Step.tsx

```diff
@@ -0,0 +1,84 @@
+import { FC, MouseEvent, ReactNode } from "react";
+
+export interface IStepEvent extends MouseEvent<HTMLElement> {
+  label?: string;
+  index?: number;
+}
+
+interface IStepProperties {
+  activeIndex: number;
+  activeStepIcon?: string | ReactNode;
+  command?: (event: IStepEvent) => void;
+  handleActiveIndex: (event: IStepEvent) => void;
+  index: number;
+  label?: string;
+  isLineDashed?: boolean;
```

Comment:
Allow room for extension. Following way, you can add new styles without needing to change much of the code.
```suggestion
  lineStyle?: 'solid' | 'dashed';
```